configure.ac

```diff
@@ -4314,47 +4314,46 @@ then
 fi
 if test "$ENABLED_STUNNEL" = "yes"
 then
-    # Requires opensslextra make sure on
```

Comment:
Why remove comments?

Reply:
They struck me as self-explanatory. If you think they add value, though, I'll add them back.

Reply:
Don't worry. I agree. but make sure there is a comment at top saying all these are needed.